### DISCUSSION TITLE
Please allow to add waveform by drag'n'drop .wav file

### DISCUSSION THREAD

**Discussion by TheShadowRunner** [SubtitleEdit/subtitleedit#4885] [Ideas]
As I know (and prefer) to generate the wav file by myself rather than use VLC.
Thanks for considering.

**Comment by niksedk** [comment]
This should already work... 
Also, you can switch to FFmpeg in Options - Settings - Waveform

**Reply by TheShadowRunner** [reply]
Sorry, my bad, you're right!
The "Save WAV" feature in VirtualDub doesn't actually generate a wav but a simple dump of the audio stream (compressed) with a .wav extension!

When I feed a proper wav it works ;)
Thanks for this wonderful soft!
